filters/tee/teeloopback.go

```diff
@@ -0,0 +1,48 @@
+package tee
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
+	teePredicate "github.com/zalando/skipper/predicates/tee"
```

Comment:
Please use all lowercase for package identifiers: `teepredicate`. See the stdlib (strconv, maphash, etc...)